test/recipes/80-test_ssl_old.t

```diff
@@ -822,4 +822,37 @@ sub testssl {
         ok(run(test([@ssltest, "-bio_pair", "-tls1", "-ct"])));
         }
     };
+
+    subtest 'Session ticket + SNI enable tests' => sub {
+	######################################################################
```

Comment:
Given the number and type of tests involved, the existing option (-no_ticket) is not sufficient, as it completely disables session tickets. The test specifically look at different combinations of server context (1 and 2) enable/disable/broken and client enable/disable.